build.sbt

```diff
@@ -68,14 +76,18 @@ lazy val anorm = project
       // New functions
       missMeth("anorm.Sql.unsafeStatement$default$2"),
       missMeth("anorm.Sql.unsafeResultSet"),
-      missMeth("anorm.Sql.unsafeStatement")),
+      missMeth("anorm.Sql.unsafeStatement"),
+      missMeth("anorm.ToStatement.contramap"),
```

Comment:
In Scala 2.11, binary compatibility for traits is the same as for interfaces in Java. The trait is compiled to a `ToStatement` interface and a `ToStatement$class` static class with method implementations. When a class extends the trait, the compiler generates the "stub" implementations to forward to the static class. If you add a method to the trait with an implementation, all classes extending that trait need to be recompiled to get the implementation.

So unfortunately it's not binary compatible, which is why I suggested an alternative option.